x-pack/plugins/ml/public/components/annotations_table/annotations_table_directive.js

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+
+
+/*
+ * Chart plotting data from a single time series, with or without model plot enabled,
+ * annotated with anomalies.
+ */
+
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+import { AnnotationsTable } from './annotations_table';
+
+import 'angular';
+
+import { uiModules } from 'ui/modules';
+const module = uiModules.get('apps/ml');
+
+import chrome from 'ui/chrome';
+const mlAnnotationsEnabled = chrome.getInjected('mlAnnotationsEnabled', false);
+
+module.directive('mlAnnotationTable', function () {
+
+  function link(scope, element) {
+    function renderReactComponent() {
+      if (typeof scope.jobs === 'undefined') {
+        return;
+      }
+
+      const props = {
+        annotations: scope.annotations,
+        jobs: scope.jobs,
+        isSingleMetricViewerLinkVisible: scope.drillDown,
+        isNumberBadgeVisible: true
+      };
+
+      ReactDOM.render(
+        React.createElement(AnnotationsTable, props),
+        element[0]
+      );
+    }
+
+    renderReactComponent();
+
+    scope.$on('render', () => {
+      renderReactComponent();
+    });
+
+    function renderFocusChart() {
+      renderReactComponent(true);
```

Comment:
Might be missing something here but where is this `true` param being used in `renderReactComponent`?

Reply:
You're right of course! Copy-pasted, refactored, missed it :) Fixed in fb52cccbf9